lib/db_connection.ex

```diff
@@ -423,6 +446,9 @@ defmodule DBConnection do
     `start_link/2` docs
     * `:timeout` - The maximum time that the caller is allowed to perform
     this operation (default: `15_000`)
+    * `:deadline` - If set, specifies absolute monotonic time in milliseconds
+    by which caller must perform operation. See `System` module documentation
+    for more information on monotonic time
```

Comment:
`default: nil` and overrides `timeout` if set

Reply:
ok how about this?

```
* `:deadline` - If set, overrides `:timeout` option and specifies absolute monotonic time in milliseconds
by which caller must perform operation. See `System` module documentation
for more information on monotonic time (default: `nil`)
```

I'll update the type to be `{:deadline, integer | nil}` too.

Reply:
👍